website/content/docs/connect/config-entries/api-gateway.mdx

```diff
@@ -329,3 +397,166 @@ Specifies the Enterprise [admin partition](/consul/docs/enterprise/admin-partiti
 
 - Default: `"default"` in Enterprise
 - Data type: string
+
+### `Listeners[].default`
+
+Specifies a block of default configurations to apply to the gateway listener. All routes attached to the listener inherit the default configurations. You can specify override configurations that have precedence over default configurations. Refer to the [`override` block](#listeners-override) for details.  
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `Listeners[].default{}.JWT`
+
+Specifies a block of default JWT verification configurations to apply to the gateway listener. You can specify override configurations that have precedence over default configurations. Refer to the [`override.JWT` block](#listeners-override) for details.
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `Listeners[].default{}.JWT{}.Providers`
+
+Specifies a list of default JWT provider configurations to apply to the gateway listener. A provider configuration contains the name of the provider and claims. You can specify override configurations that have precedence over default configurations. Refer to the [`override.JWT.Providers` block](#listeners-override-providers) for details.
+
+#### Values
+
+- Default: None
+- Data type: List of maps
+
+The following table describes the parameters you can specify in a member of the `Providers` list:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `Name` | Specifies the name of the provider. | String | None |
+| `VerifyClaim` | Specifies a list of paths and a value that define the claim. Consul verifies requests that match the claims declared in the listener JWT configuration and allow the request through the gateway. The `VerifyClaims` map specifies the following settings: <li>`Path`: List of paths to the claims</li><li>`Value`: ???</li> | Map | None |
+
+Refer to [Configure JWT verification settings](#configure-jwt-verification-settings) for an example configuration. 
+
+### `Listeners[].override`
+
+Specifies a block of configurations to apply to the gateway listener. The override settings have  precedence over the [`default` configurations](#listeners-default).
```

Comment:
Related to my above comment, the overrides here ultimately have precedence over the defaults defined on the `Gateway` resource and any JWT filter on the `HTTPRoute` resource.

Reply:
I see the block below specifically mentions the route-level configuration, so maybe it doesn't make sense to mention those here as well. Just thinking out loud 😄